clients/ops-l2os/l2os-entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -exu
+
+curl \
+    --fail \
+    --retry 10 \
+    --retry-delay 2 \
+    --retry-connrefused \
+    -X POST \
+    -H "Content-Type: application/json" \
+    --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["0x0", false],"id":1}' \
+    http://172.17.0.3:8545
+
+L2OO_ADDRESS=$(jq -r .address < /L2OutputOracle.json)
+
+exec l2os \
+    --l1-eth-rpc http://172.17.0.3:8545 \
```

Comment:
Are these guaranteed to be the same IPs on every machine we deploy this to? Is it possible to use DNS instead, or pass these in from Hive itself?

Reply:
By default docker uses the `172.17. 0.0/16` subnet range. But yeah this only works if there are no other contains occupying the IP range we use. I will try to find a better way but it seems like the networking part is usually a runtime option rather than build time, so it will require configuration on the host executing the tests.